image.yaml

```diff
@@ -7,6 +7,12 @@ from: "ubi8-minimal:latest"
 labels:
     - name: "com.redhat.component"
       value: "redhat-sso-7-sso74-openshift-rhel8-container"
+    - name: "org.jboss.product"
+      value: &product "sso"
+    - name: "org.jboss.product.version"
+      value: &product_version "7.4.0.GA"
+    - name: "org.jboss.product.sso.version"
+      value: *product_version
```

Comment:
Should the `"org.jboss.product.openjdk.version":` label be added too?

Reply:
No, that one defined is in the sso-jdk  module

Reply:
Ok, failr enough. Thanks